diff -Naru ./src/Makefile.am ../swfmill-0.3.1_new/src/Makefile.am
--- ./src/Makefile.am	2010-07-14 20:30:13.000000000 -0400
+++ ../swfmill-0.3.1_new/src/Makefile.am	2011-08-25 10:52:34.000000000 -0400
@@ -74,7 +74,7 @@
 
 $(SIMPLE_DIALECT_GENERATEDSOURCES): $(SIMPLE_DIALECT_GENERATEDSOURCES:.cpp=.xml) xslt/assemble.xsl $(SIMPLE_DIALECT_XSLTS)
 	xsltproc $(srcdir)/xslt/assemble.xsl $< > $(@:.cpp=.xsl)
-	echo "#include \"xslt/xslt.h\"" > $@
+	echo "#include \"libxslt/xslt.h\"" > $@
 	echo "const char *xslt_simple = " >> $@
 	sed -e "s/namespaces=\\\"hack\\\"/$(NAMESPACES)/" -e "s/\"/\\\\\"/g" -e "s/\(.*\)/\"\1\\\\n\"/g" $(@:.cpp=.xsl) >> $@
 	echo ";" >> $@
diff -Naru ./src/Makefile.in ../swfmill-0.3.1_new/src/Makefile.in
--- ./src/Makefile.in	2010-07-15 15:18:52.000000000 -0400
+++ ../swfmill-0.3.1_new/src/Makefile.in	2011-08-25 10:52:34.000000000 -0400
@@ -1300,7 +1300,7 @@
 
 $(SIMPLE_DIALECT_GENERATEDSOURCES): $(SIMPLE_DIALECT_GENERATEDSOURCES:.cpp=.xml) xslt/assemble.xsl $(SIMPLE_DIALECT_XSLTS)
 	xsltproc $(srcdir)/xslt/assemble.xsl $< > $(@:.cpp=.xsl)
-	echo "#include \"xslt/xslt.h\"" > $@
+	echo "#include \"libxslt/xslt.h\"" > $@
 	echo "const char *xslt_simple = " >> $@
 	sed -e "s/namespaces=\\\"hack\\\"/$(NAMESPACES)/" -e "s/\"/\\\\\"/g" -e "s/\(.*\)/\"\1\\\\n\"/g" $(@:.cpp=.xsl) >> $@
 	echo ";" >> $@
diff -Naru ./src/swfmill.cpp ../swfmill-0.3.1_new/src/swfmill.cpp
--- ./src/swfmill.cpp	2010-07-14 20:30:12.000000000 -0400
+++ ../swfmill-0.3.1_new/src/swfmill.cpp	2011-08-25 12:11:21.000000000 -0400
@@ -1,7 +1,7 @@
 #include <libexslt/exslt.h>
 #include "SWFFile.h"
 #include "swft.h"
-#include "xslt.h"
+#include "libxslt/xslt.h"
 #include <cstdlib>
 #include <cstring>
 #include <sys/types.h>
@@ -387,7 +387,8 @@
 	}
 	const char *outfile = argv[argc-1];
 	argc--;
-		
+	
+	extern const char* xslt_simple;
 	internal_stylesheet = xslt_simple;
 	
 	xmlDocPtr doc = xmlNewDoc((const xmlChar*)"1.0");
@@ -413,7 +414,7 @@
 
 int main( int argc, char *argv[] ) {
 	char *command = NULL;
-	
+	extern const char* xslt_simple;	
 	swft_register();
 	exsltRegisterAll();
 
@@ -483,6 +484,7 @@
 	} else if( !strcmp( command, "xslt" ) ) {
 		return swfmill_xslt( argc-i, &argv[i] );
 	} else if( !strcmp( command, "simple" ) ) {
+		extern const char* xslt_simple;
 		internal_stylesheet = xslt_simple;
 		return swfmill_xml2swf( argc-i, &argv[i] );
 	} else if( !strcmp( command, "library" ) ) {
